content/tracing/span/time_events/message_event.md

```diff
@@ -96,6 +96,9 @@ serverEvent = time_event.MessageEvent(seqNumber, type=time_event.Type.SENT,
                 uncompressed_size_bytes=1024, compressed_size_bytes=512)
 span.add_time_event(time_event.TimeEvent(datetime.datetime.utcnow(), serverEvent))
 {{</highlight>}}
+
+{{<highlight js>}}
```

Comment:
I guess this means message events are unimplemented?

Reply:
Thanks for commenting on this. At first glance looking through the node repo I didn't see anything about message events, but I just looked again and found a bunch of info. I'll push up another commit in a few minutes.

Reply:
Actually, upon looking at it again, I don't see anywhere to instantiate a message event. I only see [references](https://github.com/census-instrumentation/opencensus-node/blob/59bd2161ceeb60ddb8861f38675f42409088021e/packages/opencensus-core/src/trace/model/span-base.ts#L178-L184).

Reply:
Ill remove the js snippet for now